src/panel/poi/blocks/Covid19.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { Fragment } from 'react';
+import TimeTable from './TimeTable';
+import covidStrings from './covid_strings';
+import OsmSchedule from 'src/adapters/osm_schedule';
+import OpeningHour from 'src/components/OpeningHour';
+
+// @TODO: refacto OsmSchedule so it doesn't need presentational data
+const scheduleMessages = {
+  open: {
+    msg: 'Ouvert',
+    color: '#60ad51',
+  },
+  closed: {
+    msg: 'Fermé',
+    color: '#8c0212',
+  },
+};
+
+const getContent = (poi, { status, opening_hours, note, contribute_url }) => {
+  const additionalInfo = note &&
+    <div className="covid19-note">
+      <i className="icon-icon_info" />
+      <span dangerouslySetInnerHTML={{ __html: note }}/>
+    </div>;
+
+  const source =
+    <a className="covid19-contributeLink" href={contribute_url}>
+      {covidStrings.linkToCaResteOuvert}
+    </a>;
+
+  let content;
+  let schedule;
+  switch (status) {
+  case 'open':
+  case 'open_as_usual':
+    schedule = opening_hours && new OsmSchedule(opening_hours, scheduleMessages);
+    content = <Fragment>
+      <div className="covid19-status covid19-status--open">{covidStrings.statusOpen}</div>
+      {schedule && <div className="covid19-timeTableContainer">
+        <i className="icon-icon_clock" />
+        <TimeTable title={<OpeningHour poi={poi} />} schedule={schedule} />
+      </div>}
+      {!schedule && <div className="covid19-changeWarning">{covidStrings.hoursMayChange}</div>}
+      {additionalInfo}
+      {source}
```

Comment:
The link to caresteouvert.fr may be useful independently from the status (and if`contribute_url` is not null), especially if no data is available

Reply:
I agree with you and that's the topic of ongoing discussions… I'll try to expand the cases where we can promote contribution.